js/woo.js

```diff
@@ -1375,6 +1380,44 @@ module.exports = class woo extends Exchange {
         return this.safeString (statuses, status, status);
     }
 
+    async transfer (code, amount, fromAccount, toAccount, params = {}) {
+        await this.loadMarkets ();
+        const currency = this.currency (code);
+        const request = {
+            'token': currency['id'],
+            'amount': this.parseNumber (amount),
+            'from_application_id': fromAccount,
+            'to_application_id': toAccount,
+        };
+        const response = await this.v1PrivatePostAssetMainSubTransfer (this.extend (request, params));
+        //
+        //     {
+        //         "success": true,
+        //         "id": 200
+        //     }
+        //
+        const transferOptions = this.safeValue (this.options, 'transfer', {});
+        const fillResponseFromRequest = this.safeValue (transferOptions, 'fillResponseFromRequest', true);
+        const transfer = {
```

Comment:
Normally this would go inside `parseTransfer` however `parseTransfer` is already defined for a different response.

Reply:
Can we still move this parsing but to `parseTransfer` ? We have other samples where we parse differing structures with the same parser, for example: https://github.com/ccxt/ccxt/blob/master/js/woo.js#L401